src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -67,4 +67,9 @@ public CommandResult execute() {
         }
     }
 
+    @Override
```

Comment:
Header comment is necessary for overriden methods to inform the others what the overriding is for. Add the appropriate comment for the isMutating() method. Same for below